applications/acdc/test/test.config

```diff
@@ -0,0 +1,10 @@
+[
+ {lager, [
+          {handlers, [
+                      {lager_console_backend, '=info'}
```

Comment:
I couldn't get this to work (see https://github.com/2600hz/kazoo/compare/test-config )
Do you think it is due to eunit owning the process group?
Is it possible to redirect to group `user` with `lager`?